src/Lykke.MatchingEngine.Connector/Domain/Fee.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Lykke.MatchingEngine.Connector.Abstractions.Models;
+
+namespace Lykke.MatchingEngine.Connector.Domain
+{
+    public class Fee
```

Comment:
Better to make this class immutable. Moreover, consider to move methods of the FeeCalculationService class here, as static factory methods. Right now this class looks like plain dto, not a domain model